removing mentions of irc and telegram, and formatting with prettier

bigwavedave42 opened the pull request.

bigwavedave42 committed:
removing mentions of irc and telegram, and formatting with prettier

Signed-off-by: bigwavedave42 <bigwavedave@gmail.com>

BKSteve commented on Home.md:
I'd put the discord link to chat leaving discussions as the text. Wasn't sure if this would work but we can't always send people to the invitation to discord server.
[Discussions](https://discord.com/channels/502612977271439372/502612977803984898)

bigwavedave42 replied:
Understood.  I was trying to streamline to have 1 link into Discord, making it easier for any find/replace we might have to do in the future... and for consistency.  I get your point though and have updated accordingly.

bigwavedave42 committed:
Merge branch 'SickChill:main' into main

bigwavedave42 committed:
Updating Discord link to go direct to Discussions

bigwavedave42 committed:
Merge branch 'SickChill:main' into main

miigotu commented on Home.md:
```diff
@@ -6,7 +6,7 @@ On the right side of the page you see the available wiki pages. For new users, y
 If you encounter a bug, please goto [Discord](https://discord.com/invite/U8WPBdf) and open a Chat, Help or Issue.
 
-These documents are a work-in-progress. If you'd like to contribute, please get in touch with us on [Discord](https://discord.com/invite/U8WPBdf), or [Discussions](https://discord.com/channels/502612977271439372/502612977803984898), [Telegram](https://t.me/sickchill), [IRC](https://kiwiirc.com/client/irc.freenode.net/?theme=basic#sickchill) or open an [issue here on GitHub.](https://github.com/Si ...
+These documents are a work-in-progress. If you'd like to contribute, please get in touch with us on Discord in [Discussions](https://discord.com/channels/502612977271439372/502612977803984898), or open an [issue here on GitHub.](https://github.com/SickChill/SickChill/issues)
```
The part about opening an issue on GitHub can go away, we do issues on discord now

miigotu replied:
Also, I think this was the part you could put both a link to the discord invite and the discussion. Two links, one with Discussions and the other with (Invite) or something.

bigwavedave42 replied:
> The part about opening an issue on GitHub can go away, we do issues on discord now

Done.  Might need to look for similar references elsewhere in the wiki.

> Also, I think this was the part you could put both a link to the discord invite and the discussion. Two links, one with Discussions and the other with (Invite) or something.

Done.  Took a stab at that.

bigwavedave42 replied:
Found additional references and tried to clean that up as well.

bigwavedave42 committed:
add links to both the Discord invite and discussion

bigwavedave42 committed:
Removing Reference to TigerVPN and spelling/grammar

Asked to remove the reference to TigerVPN, since there isn't a relationship and it might imply a recommendation.  No specific concerns implied, just made it more generic.

bigwavedave42 committed:
Replacing reference to GitHub issues

Per miigotu request, removing reference to GitHub Issues posting in favor of utilizing Discord.

bigwavedave42 committed:
Removing GitHub Issues reference and spelling/grammar

bigwavedave42 committed:
Removing reference to GitHub Issues

bigwavedave42 committed:
Remove GitHub Issues and update spelling/grammar
